geoviews/plotting/mpl/__init__.py

```diff
@@ -288,6 +287,24 @@ def update_handles(self, *args):
         return GeoPlot.update_handles(self, *args)
 
 
+class GeoImageStackPlot(GeoImagePlot):
+
+    style_opts = ['alpha', 'cmap', 'visible', 'filterrad', 'clims', 'norm']
+
+    def __init__(self, element, **params):
+        super().__init__(element, **params)
+
+    def get_data(self, element, ranges, style):
+        self._norm_kwargs(element, ranges, style, element.vdims[0])
+        style.pop('interpolation', None)
+        xs, ys, zs = geo_mesh(element)
+        xs = GridInterface._infer_interval_breaks(xs)
+        ys = GridInterface._infer_interval_breaks(ys)
+        if self.geographic:
+            style['transform'] = element.crs
+        return (xs, ys, zs), style, {}
```

Comment:
I have no idea why MPL ImageStack is not going through this code; if I add print/raise, it doesn't do anything.